src/pages/sections/Welcome/Welcome.tsx

```diff
@@ -0,0 +1,25 @@
+import './Welcome.scss';
+
+export const Welcome = () => (
```

Comment:
Let's support 320px as the smallest screen. That is probably the standard nowadays.

![CleanShot 2024-11-05 at 21 03 25](https://github.com/user-attachments/assets/5175c814-3731-46eb-99cc-0732141953ab)


Unfortunately, the web breaks even earlier